common/lib/xmodule/xmodule/js/main_requirejs.js

```diff
@@ -0,0 +1,8 @@
+(function(requirejs) {
+    requirejs.config({
+        paths: {
+            "moment": "xmodule/include/common_static/js/vendor/moment.min"
```

Comment:
The JS in common just depends on the application using it to load all of its dependencies. So for LMS, the dependencies are in /edx-platform/lms/static/lms/js/require-config.js, and for cms, /edx-platform/cms/static/cms/js/require-config.js.

Both of those already list moment. @waheedahmed does it work if you just declare the video dependency on moment in 01_initialize.js and leave out main_requirejs.js?